lib/python/picongpu/picmi/interaction/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .interactioninterface import InteractionInterface
+from .interaction import Interaction
+from . import ionization
+
+__all__ = ["InteractionInterface", "Interaction", "ionization"]
```

Comment:
I think as long as you're not changing the default behaviour, adding this line is a maintenance burden. I'd suggest to add it later if ever needed. Applies to all (but I won't tag it everywhere).

Reply:
will be fixed in a future PR